utils-metrics/src/main/scala/org/apache/spark/rdd/InstrumentedOrderedRDDFunctions.scala

```diff
@@ -28,7 +28,7 @@ import scala.reflect.ClassTag
  */
 class InstrumentedOrderedRDDFunctions[K: Ordering: ClassTag, V: ClassTag](self: RDD[(K, V)]) extends Serializable {
 
-  def sortByKey(ascending: Boolean = true, numPartitions: Int = self.partitions.size): RDD[(K, V)] = self match {
+  def sortByKey(ascending: Boolean = true, numPartitions: Int = self.partitions.length): RDD[(K, V)] = self match {
```

Comment:
I'm neutral on this one. I think `size` is defined for all `Iterable`s in Scala, while `length` is not.